Doc/library/asyncio-task.rst

```diff
@@ -528,6 +528,28 @@ Task functions
    the event loop object used by the underlying task or coroutine.  If it's
    not provided, the default event loop is used.
 
+
+.. function:: current_task(loop=None):
+
+   Return a currently executed task or ``None`` if no task is running.
+
+   If *loop* is ``None`` :func:`get_running_loop` is used for gettung
+   current loop.
```

Comment:
gettung -> getting

Please change to "If *loop* is ``None``, :func:`get_running_loop` is used to get the current loop."